hack/cluster-openshift.sh

```diff
@@ -523,8 +523,14 @@ if [ "$_CMD" = "up" ]; then
     if [ "${ISTIO_ENABLED}" == "true" ] ; then
       if [ "${_CREATE_INSTALLATION_RESOURCE}" == "true" ] ; then
         echo "Installing Istio via Installation Custom Resource"
-        debug "${MAISTRA_ISTIO_OC_COMMAND} create -n istio-operator -f ${MAISTRA_INSTALL_YAML}"
-        ${MAISTRA_ISTIO_OC_COMMAND} create -n istio-operator -f ${MAISTRA_INSTALL_YAML}
+        echo "Creating a istio-system namespace"
+        debug "${MAISTRA_ISTIO_OC_COMMAND} new-project istio-system"
+        ${MAISTRA_ISTIO_OC_COMMAND} new-project istio-system
+        # Note, Maistra requires to install the CR in the target namespace not in the operator one
+        # https://maistra.io/docs/getting_started/install/
+        echo "Creating a CR under istio-system namespace"
+        debug "${MAISTRA_ISTIO_OC_COMMAND} create -n istio-system -f ${MAISTRA_INSTALL_YAML}"
+        ${MAISTRA_ISTIO_OC_COMMAND} create -n istio-system -f ${MAISTRA_INSTALL_YAML}
```

Comment:
based on my other comment, this line should change to cat the YAML and pass it through envsubst - that way, we can customize it based on options the user passes to the cluster-openshift.sh script (such as the --no-jaeger option).